### DISCUSSION TITLE
Guidance on caching

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#13634] [Request Help]
I currently use the file system based caching, but it's gone up to close to a Gig now for 86 projects, so I'm not sure whether any pruning is going on there, and I want to consider my options.

Are there any guidelines on what needs to be configured for the Redis cache for self hosted Renovate instances? I see just a URL option in documentation, so I'm guessing there's no authentication going on for Redis?

**Comment by rarkins** [comment]
1GB is not something I'd be worried about, or something we'd consider a "problem". Renovate is regularly used on projects which themselves take up gigabytes each once installed.

I think - but not certain - that you can embed authentication in the Redis URL
